src/test/java/org/folio/consortia/service/TenantServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.consortia.service;
+
+import org.folio.consortia.repository.TenantRepository;
+import org.folio.consortia.repository.entity.Tenant;
+import org.folio.consortia.service.impl.TenantServiceImpl;
+import org.folio.pv.domain.dto.TenantCollection;
+import org.folio.spring.data.OffsetRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.batch.BatchAutoConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.PageImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SpringBootTest
+@EnableAutoConfiguration(exclude = BatchAutoConfiguration.class)
+@EntityScan(basePackageClasses = Tenant.class)
+class TenantServiceTest {
+  @InjectMocks
+  private TenantServiceImpl tenantService;
+  @Mock
+  private TenantRepository repository;
+
+  @Test
+  void shouldGetTenantList()
+  {
```

Comment:
lets use the same code style with { } , so move them on previous line

Reply:
fixed